output.go

```diff
@@ -46,7 +46,28 @@ func Output(w io.Writer, g *Generator, pkg string) {
 
 	for _, k := range getOrderedStructNames(structs) {
 		s := structs[k]
-		if s.GenerateCode {
+		generateCode := false
+		if !skipValidationFlag {
+			for _, f := range s.Fields {
+				if f.Required {
+					generateCode = true
+					break
+				}
+			}
+
+			// No aditional properties are allowed
+			if s.AdditionalType == "false" {
+				generateCode = true
+			}
+		}
+
+		// If the struct has an additionalProperties we need to generate
+		// the Marshal/Unmarshal code for handling them
+		if s.AdditionalType != "" && s.AdditionalType != "false" {
```

Comment:
If additionalType is not present we default to `true`. Is this intended?

Reply:
If `AdditionalType` is empty (I'm guessing you mean that when saying is not present) then `if s.AdditionalType != ""` would be false, we won't go inside the if and `generateCode` should be false. Am I missing something here?